v1/pipeline/steps/cd/values-manifest.yaml

```diff
@@ -7,6 +7,11 @@ allOf:
       type: string
     store:
       $ref: store-config-wrapper.yaml
+    optionalValuesYaml:
+      oneOf:
+      - $ref: parameter-field-boolean.yaml
+      - type: string
```

Comment:
instead of string we should also add pattern to support only expressions and input sets:
```
pattern: (<\+.+>.*)
minLength: 1
```